libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -681,6 +701,15 @@ bool AP_Follow::handle_follow_target_message(const mavlink_message_t &msg)
         return false;
     }
 
+    // apply jitter-corrected timestamp to this update
```

Comment:
This needs to be moved down where it was set before. It can be updated if the message is not valid.